CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
+* [CHANGE] Configuration format changed and switched to YAML.
+* [ENHANCEMENT] Redesign of state-preserving target discovery.
+* [ENHANCEMENT] Specify scrape URL scheme and basic HTTP auth for non-static targets.
+* [FEATURE] Attach meaningful labels to targets via relabeling.
+* [FEATURE] Configuration/rule reloading at runtime.
+* [FEATURE] Target discovery via file watches.
+* [FEATURE] Target discovery via Consul.
+* [ENHANCEMENT] Simplified binary operation evaluation.
+* [ENHANCEMENT] More stable component initialization.
+* [FEATURE] Added internal testing language.
```

Comment:
Strictly speaking its not a QL testing language. It's an evaluation/expression testing language. The query _language_ is gone after parsing.

At least that's how I would define it.